docker/Dockerfile.relay-server2

```diff
@@ -0,0 +1,12 @@
+FROM openjdk:11
+
+RUN apt-get update \
+    && apt-get install -y time
+
+COPY logging.xml /root
+COPY flight-server-all.jar /root/
+COPY run_relay_server2.sh /root/
```

Comment:
It is unclear what the difference is between `run_relay_server.sh` and `run_relay_server2.sh`. Please rename